### PR TITLE
Add Task feature

### PR DESCRIPTION
**Pull request by varun-Tad**
I have created the Homepage and add task feature
Files to review
1.Homepage.js
2.Homepage.css
3.Taskpage.js
4.Taskpage.css

Also please review the Cancel button and the Add Task button in the modal which appear when you click on the Add Task button. Please ignore the edit/delete& timer button. I am yet to implement those features

Preview: https://tourmaline-kleicha-a70cdd.netlify.app/

### CODE REVIEW

File: timeit/src/Pages/HomePage/Homepage.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+import "./Homepage.css";
+
+const Homepage = () => {
+  const [enteredName, setEnteredName] = useState("");
+  const nameChangeHandler = (e) => {
+    setEnteredName(e.target.value);
+  };
+
+  const clickHandler = () => {
+    console.log(enteredName);
```
Comment (kaustubh-jsr) [inline timeit/src/Pages/HomePage/Homepage.js]: You can remove this console log

**Review by kaustubh-jsr — APPROVED** [review]
The code is clean, could not find any issues with it. You can alter the UI to make the input centered, if that is intended, if not please ignore this one
pr review
.

File: timeit/src/Pages/TaskPage/Taskpage.js [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+                }
+                setTitle("");
+              }}
+            >
+              Add Task
+            </button>
+          </div>
+        </form>
+      )}
+      {isModal || isModal}
```
Comment (anasm11) [inline timeit/src/Pages/TaskPage/Taskpage.js]: What does this line do? I think it can be removed.

**Review by anasm11 — APPROVED** [review]
The code looks good. You can merge.

File: timeit/src/Pages/TaskPage/Taskpage.js (hunk 2 above)
Comment (varun-Tad) [reply timeit/src/Pages/TaskPage/Taskpage.js]: yes my bad i forgot to remove it.

**Pull request merged**